src/common/common.cpp

```diff
@@ -310,6 +310,33 @@ bool run_non_elevated(const std::wstring& file, const std::wstring& params) {
   return succedded;
 }
 
+bool run_same_elevation(const std::wstring& file, const std::wstring& params) {
```

Comment:
Make sure to run the .clang-format

Reply:
It looks like the file was not yet formatted.